sencha-workspace/SlateAdmin/app/Application.js

```diff
@@ -7,9 +7,6 @@ Ext.define('SlateAdmin.Application', {
     requires: [
         'SlateAdmin.API',
 
-        // Jarvus enhancements
-        'Jarvus.ext.override.data.RequireLoadedStores',
```

Comment:
@BillClinton this require shouldn't be removed.

Hotfix requires should be removed as the app should only have to pull in sencha-hotfixes and not have to worry about framework bugs.

This override though is a new feature that the app depends on, and we want the app to fail to load and fail to build if it is not present because its package failed to load for whatever reason. Try to require feature overrides from packages as close as possible to where they are used. In cases like this where it's an override that is used extensively throughout the app this is a good place for it